api/proto/teleport/legacy/types/types.proto

```diff
@@ -5190,13 +5190,8 @@ message PluginBearerTokenCredentials {
   // Token is the literal bearer token to be submitted to the 3rd-party API provider.
   string token = 1;
 
-  // TokenFile is a path to the local file containing a bearer token.
```

Comment:
Are we sure we want reserve this and not deprecate it instead?

Reply:
It seems like we usually deprecate first, then remove the field + reserve.
https://github.com/gravitational/teleport/commit/85c426bbcd39096bb60c6ee97acb0ae0c0b5fa42

Since this field is not deeply entrenched in the codebase yet, and we're removing all usages immediately, it seems that reserved might be more fitting (can't accidentally use this field again in new code).